src/praisonai-agents/tests/unit/tools/test_async_tool_retry_parity.py

```
@@ -102,4 +102,43 @@ def test_async_honours_retryable_tool_execution_error():
+    calls = []
+
+    def upstream(x: str = "") -> dict:
+        calls.append(1)
+        return {"error": "upstream 503", "retryable": True}
+
+    try:
+        Agent(
+            name="t", instructions="x", tools=[upstream], tool_config=FAST
+        ).execute_tool("upstream", {})
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)